### PR TITLE
PG-1206 Reworked intro to pg_tde

### PR DESCRIPTION
**Pull request by nastena1606**
PG-1206

### Description
Reworked the intro on the index page to make it more appealing. 
Added a link to what's TDE. 
Added some FAQ

### CODE REVIEW

File: documentation/docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Why do I need TDE?
+
+- Compliance to security and legal regulations like GDPR, PCI DSS and others
+- Encryption of backups
+- Granular encryption of specific data sets and reducing the performance overhead that encryption brings
+- Additional layer of security to existing security measures
+
+## I use disk-level encryption. Why should I care about TDE?
+
+Encrypting hard drive encrypts all data, system and application files that are there. However, disk encryption doesn’t protect your data after the boot-up of your system. During runtime, the files are decrypted with disk-encryption.
```
Comment (ImTheKai) [inline documentation/docs/faq.md:12]: "Encrypting a hard drive..." or "Encrypting hard drives.." also "..encrypts all data, including system and..." (that is easier to read I think).

File: documentation/docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+TDE focuses specifically on data files and offers a more granular control over encrypted data. It also ensures that files are encrypted on disk during runtime and when moved to another system or storage.
+
+## Is TDE enough to ensure data security?
+
+No. TDE is an additional layer to ensure data security. It protects data at rest. Consider introducing also these measures:
+
+* Access control and authentication
+* Strong network security like TLS
+* Regular monitoring and auditing
+* Additional data protection for sensitive fields (e.g., application-layer encryption)
```
Comment (ImTheKai) [inline documentation/docs/faq.md:23]: I would add storage/disk encryption here as well.

**Review by ImTheKai — APPROVED** [review]

File: documentation/docs/faq.md (hunk 2 above)
Comment (nastena1606) [reply documentation/docs/faq.md:26]: Added

File: documentation/docs/index.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # `pg_tde` documentation
 
-`pg_tde` is the extension that brings in [Transparent Data Encryption (TDE)](tde.md) to PostgreSQL and enables users to keep sensitive data safe and secure. The encryption is transparent for users allowing them to access and manipulate the data and not to worry about the encryption process.
+`pg_tde` is the open source PostgreSQL extension that provides Transparent Data Encryption (TDE) to protect data at rest. This ensures that the data stored on disk is encrypted, and no one can read it without the proper encryption keys, even if they gain access to the physical storage media. 
 
-Users can configure encryption differently for each database, encrypting specific tables in some databases with different encryption keys, while keeping others non encrypted. 
+You can configure encryption differently for each database, encrypting specific tables in some databases with different encryption keys while keeping others unencrypted.
+
+Lear more [what is Transparent Data Encryption](tde.md#how-does-it-work) and [why you need it](tde.md#why-do-you-need-tde).
```
Comment (nastena1606) [inline documentation/docs/index.md:7]:
```suggestion
Learn more [what is Transparent Data Encryption](tde.md#how-does-it-work) and [why you need it](tde.md#why-do-you-need-tde).
```

File: documentation/docs/tde.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-Transparent Data Encryption offers encryption at the file level and solves the problem of protecting data at rest. The encryption is transparent for users allowing them to access and manipulate the data and not to worry about the encryption process.
+Transparent Data Encryption is a technology to protect data at rest. The encryption process happens transparently in the background, without affecting database operations. Data is automatically encrypted as it's written to the disk and decrypted as it's read, all in real-time. Users and applications interact with the data as usual without noticing any difference.
 
 ## How does it work?
 
 To encrypt the data, two types of keys are used:
 
-* Database keys to encrypt user data. These are stored internally, near the data that they encrypt.
+* Table encryption keys (TEK) to encrypt user data. These keys are stored internally, near the data that they encrypt.
 * The principal key to encrypt database keys. It is kept separately from the database keys and is managed externally.
```
Comment (artemgavrilov) [inline documentation/docs/tde.md:10]:
```suggestion
* The principal key to encrypt table keys. It is kept separately from the table keys and is managed externally. 
```

File: documentation/docs/tde.md [hunk 5]
```diff
@@ -27,13 +27,13 @@ Using TDE has the following benefits:
...
-    - Ensure data safety when at rest and in motion
-    - Comply with security standards like HIPAA, PCI DSS, SOC 2, ISO 27001
+    - Ensure data safety when it is stored on disk and in backups
+    - Comply with security and legal standards like HIPAA, PCI DSS, SOC 2, ISO 27001
 
 * For DBAs:
    
-    - Allows defining what to encrypt in the table and with what key
-    - Encryption on storage level is not a must to provide data safety. However, using TDE and storage-level encryption together adds another layer of data security
+    - Granular encryption of specific tablespaces and reducing the performance overhead that encryption brings
```
Comment (artemgavrilov) [inline documentation/docs/tde.md:35]:
```suggestion
    - Granular encryption of specific tables and reducing the performance overhead that encryption brings
```

**Pull request merged**